test/unit/preact/test-base-element-mapping.js

```diff
@@ -97,6 +98,7 @@ describes.realWin('PreactBaseElement', {amp: true}, (env) => {
           unknown="1"
           part-a="A"
           part-b="B"
+          data-param-test="helloworld"
```

Comment:
Let's add one more `data-param-x`. Also, let's make it with dashes, e.g. `data-param-test-two` so that we can explicitly canonize the decision here.